alert/HANDLERS.md

```diff
@@ -208,6 +208,7 @@ Add the following snippet to the `service.go` file.
 // Handle takes an event and posts its message to the Foo service chat room.
 func (h *handler) Handle(event alert.Event) {
 	if err := h.s.Alert(h.c.Room, event.State.Message); err != nil {
+		h.incrementErrorCount()
```

Comment:
This doesn't look right as the handler is not a node and so will not have the incrementErrorCount method.